src/session_manager.rs

```diff
@@ -698,14 +695,15 @@ impl SessionManager {
 
         // Enable notifications for this client
         RUNTIME.spawn(async move {
-            functions::SetOption::new()
-                .name("notification_group_count_max".to_string())
-                .value(enums::OptionValue::Integer(types::OptionValueInteger {
+            functions::set_option(
+                "notification_group_count_max".to_string(),
```

Comment:
We should be able to use &str in these cases.

Reply:
Actually, we rather not, it's a bit difficult to predict when we need string literals or not in these cases.